src/client/datascience/commands/notebookCommands.ts

```diff
@@ -48,17 +44,8 @@ export class NotebookCommands implements IDisposable {
     }
 
     private toggleOutput() {
-        if (this.useNativeNotebook && this.notebook.activeNotebookEditor?.selections.length) {
-            const cells = this.notebook.activeNotebookEditor.document.getCells(
-                this.notebook.activeNotebookEditor?.selections[0]
-            );
-            if (cells.length) {
-                const cmd =
-                    cells[0].metadata.outputCollapsed === true
-                        ? 'notebook.cell.expandCellOutput'
-                        : 'notebook.cell.collapseCellOutput';
-                this.commandManager.executeCommand(cmd).then(noop, noop);
-            }
+        if (this.notebookEditorProvider.activeEditor?.toggleOutput) {
+            this.notebookEditorProvider.activeEditor.toggleOutput();
```

Comment:
I think its cleaner to have it outside this class (that's what we do for other code).